BrainPortal/app/views/demands/show.html.erb

```diff
@@ -0,0 +1,188 @@
+<%
+#
+# CBRAIN Project
+#
+# Copyright (C) 2008-2012
+# The Royal Institution for the Advancement of Learning
+# McGill University
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+%>
+
+<% title 'Registration Info' %>
+
+<% if @exception_trace.present? %>
+  <div class="flash_error"><pre><%= @exception_trace %></pre></div>
+<% end %>
+
+<h1>Account Request Summary</h1>
+
+<% if !current_user.nil? %>
+  <% if current_user.has_role?(:admin_user) %>
+    <h3>Editing this form:</h3>
+      As an administrator, you can <%= link_to "edit", :action => :edit %> this form.
+
+      <% if ! @demand.approved_by.present? %>
```

Comment:
This page allowed me to click 'approve' even though there was a username conflict with an existing user (this was show in the index page).